src/Microsoft.Health.Fhir.Shared.Web/appsettings.json

```diff
@@ -152,6 +152,8 @@
     "CosmosDb": {
         "Host": null,
         "Key": null,
+        "UseManagedIdentity": false,
```

Comment:
Just checking default for UseManagedIdentity is false?
Do we need to also add AllowCollectionSetup?

Reply:
Added a comment, `"UseManagedIdentity": false` will continue to work with local emulator